extensions/python/src/test/java/com/hazelcast/jet/python/PythonServiceTest.java

```diff
@@ -49,7 +49,12 @@
     private static final int ITEM_COUNT = 10_000;
     private static final String ECHO_HANDLER_FUNCTION =
             "def handle(input_list):\n" +
-            "    return ['echo-%s' % i for i in input_list]\n";
+ "    return ['echo-%s' % i for i in input_list]\n";
+
+    private static final String FAILING_FUNCTION
+            = "def handle(input_list):\n"
+            + "    assert 1 == 2\n"
+            + "    return ['echo-%s' % i for i in input_list]\n";
```

Comment:
This line probably isn't necessary. Also can't assertions be turned off as in java?